src/opt/memory_coloring_impl.cpp

```diff
@@ -249,8 +249,8 @@ void memory_coloring_impl::verify()
 
             if(segment.begin == invalid_offset)
             {
-                if(!interval.is_live_on_entry)
-                    MIGRAPHX_THROW("interval is not live on entry");
+                // if(!interval.is_live_on_entry)
```

Comment:
can these comments be removed?

Reply:
I am not sure, this seem to error on some cases, but I am not sure if these checks are needed or not.

Reply:
right now it's commented out. Is it still throwing an error if deleted? Or are you saying we should keep these for reference in case we need to activate these checks in the future? If that's the case, maybe we should just add a comment to explain this.